en/middleware/xrce_dds.md

```diff
@@ -0,0 +1,303 @@
+# XRCE-DDS (PX4-FastDDS Bridge)
+
+:::note
+XRCE-DDS replaces the [Fast-RTPS Bridge](https://docs.px4.io/v1.13/en/middleware/micrortps.html#rtps-dds-interface-px4-fast-rtps-dds-bridge) used in PX4 v1.13.
+:::
+
+PX4 uses XRCE-DDS middlware to allow [uORB messages](../middleware/uorb.md) to be published and subscribed on a companion computer as though they were ROS 2 topics.
+This provides a fast and reliable integration between PX4 and ROS2, and makes it much easier for ROS 2 applications to get vehicle information and send commands. 
+
+PX4 uses an XRCE-DDS implementation that leverages [eProsima Micro XRCE-DDS](https://micro-xrce-dds.docs.eprosima.com/en/stable/introduction.html).
+
+The following guide describes the architecture and various options for setting up the client and agent.
+In particular it covers the options that are most important to PX4 users.
+
```

Comment:
FYI Point  above is that this is not an exhaustive guide - for that you see the eProsima guide. But we don't do everything the architecture allows - e.g. connection over CAN, so this is "PX4s take on the infrastructure".